sourmash_lib/_minhash.pyx

```diff
@@ -323,9 +322,12 @@ cdef class MinHash(object):
             common = set(self.get_mins())
             common.intersection_update(other.get_mins())
             common.intersection_update(combined_mh.mins)
-            n = len(common)
 
-        size = max(combined_mh.size(), 1)
+        return common, max(combined_mh.size(), 1)
```

Comment:
This is a weird return value (if we think of MinHashes as sets), but `compare` needs the size of the combined MinHash...